store/inmem/InMem.go

```diff
@@ -37,77 +37,49 @@ func ProvideInMem() store.S {
 
 func (i *InMem) Push(key model.Key, item store.OwnableItem) error {
 	i.lock.Lock()
-	if _, ok := i.data[key.Bucket]; !ok {
-		i.data[key.Bucket] = map[string]store.OwnableItem{
-			key.ID: item,
-		}
-	} else {
-		i.data[key.Bucket][key.ID] = item
+	defer i.lock.Unlock()
+	if i.data[key.Bucket] == nil {
+		i.data[key.Bucket] = map[string]store.OwnableItem{}
 	}
-	i.lock.Unlock()
+	i.data[key.Bucket][key.ID] = item
 	return nil
 }
 
 func (i *InMem) Get(key model.Key) (store.OwnableItem, error) {
-	var (
-		item store.OwnableItem
-		err  error
-	)
 	i.lock.RLock()
 	defer i.lock.RUnlock()
-	if _, ok := i.data[key.Bucket]; !ok {
-		err = store.ErrBucketNotFound
-	} else {
-		if value, ok := i.data[key.Bucket][key.ID]; !ok {
-			err = store.ErrItemNotFound
-		} else {
-			item = value
-		}
+	bucket, ok := i.data[key.Bucket]
+	if !ok {
+		return store.OwnableItem{}, store.SanitizeError(store.ItemOperationError{Err: store.ErrItemNotFound, Key: key, Operation: "get"})
 	}
-	if err != nil {
-		err = store.ItemOperationError{Err: err, Key: key, Operation: "get"}
+	item, ok := bucket[key.ID]
+	if !ok {
+		return store.OwnableItem{}, store.SanitizeError(store.ItemOperationError{Err: store.ErrItemNotFound, Key: key, Operation: "get"})
```

Comment:
I understand from a program logic perspective, we don't care which case causes this error.  But is there a reason that a human might want to know when debugging? Is there a benefit of wrapping the `store.ErrItemNotFound` error with some text meant for humans to read in logs saying either the bucket or item didn't exist? ie:
```go
return store.OwnableItem{}, store.SanitizeError(store.ItemOperationError{
    Err: fmt.Errorf("%w: no bucket", store.ErrItemNotFound), 
    Key: key, Operation: "get",
})
```

Reply:
I don't think so. In either case, the only action to remediate it is actually creating the item.